src/common/configuration.ts

```diff
@@ -0,0 +1,72 @@
+import { ConfigurationChangeEvent, WorkspaceConfiguration } from 'vscode'
+import { ScanEngine } from './constants'
+
+export const CONFIGURATION_IDENTIFIER = 'spectral'
+export const USE_IAC_ENGINE_SETTING = 'scan.engines.useIacEngine'
+export const USE_OSS_ENGINE_SETTING = 'scan.engines.useOssEngine'
```

Comment:
these const should be on the constants file

Reply:
fixed, https://github.com/SpectralOps/vscode-extension/pull/7/commits/e1fded33e247a3fd2e4a7e247f1b8c1868623af6